### PR TITLE
Add back hostname script to DHCP exit hooks.

### PR DESCRIPTION
**Pull request by zmarano**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+google-compute-image-packages (2.6.0-2) stable; urgency=low
```
Comment (illfelder) [inline debian/changelog]: Move the version bump to a separate pull request.

**Comment by palladius** [comment]
Can you please explain why you did this change? It broke some customers workloads by rewriting `/etc/hosts`. I think this is a dangerous CL. Any chance we can revert it or make it togglable?
